audio/sonic/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+name                sonic
+github.setup        espeak-ng sonic 0.2.0 release-
+version             0.2.0
+revision            1
+categories          audio
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3+
+
+description         Simple library to speed up or slow down speech
+
+long_description    Simple library to speed up or slow down speech
+
+homepage            https://github.com/espeak-ng/sonic
+distname            release-${version}
+checksums           rmd160  fd728ede06bf9f2b5ff8257455c506341e5ee620 \
+                    sha256  66ae97ccd39450734a3819c342683fdb1698eb2322c011b95fcf7e1959caa1db \
+                    size    5340290
```

Comment:
Please use `github.tarball_from archive`. This will require changing the checksums as well.